upgrade-notes/storefront_20240711_123953.md

```diff
@@ -0,0 +1,3 @@
+#### refactored form validations ([#3263](https://github.com/shopsys/shopsys/pull/3263))
+
+-   refactor and unification form validations.
```

Comment:
This should also tell developers what they should do in their code, such as:

- modify new validation rules (add their names) so that it matches your wanted validation logic
- use new validation rules in other forms you might have to fully utilize the new code
- etc.